lib/ui/send/usecases/send_token_use_case.dart

```diff
@@ -33,7 +33,7 @@ class SendTokenUseCase {
         ),
       ),
       memo: memo ?? '',
-      network: Networks.telos,
+      network: Network.telos,
```

Comment:
We can fix this later but yeah this should use the user's network for sending (or doing any transactions)